asm/istio/istiod-service.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This YAML is intended to create an istiod service for validation to
+# work around istio/istio/issues/27501 in the case of no non-revision
+# installation.
+apiVersion: v1
+kind: Service
+metadata:
+  name: istiod
+  namespace: istio-system
+  labels:
+    istio.io/rev: asm-173-3 # {"$ref":"#/definitions/io.k8s.cli.setters.anthos.servicemesh.rev"}
```

Comment:
Any reason why we removed the operator labels?
```
    operator.istio.io/component: Pilot
    operator.istio.io/managed: Reconcile
    operator.istio.io/version: 1.7.3-asm.6
```

btw, operator labels now are used for pruning resources not used in a new version, which i think it is the right to not adding them here.